AudibleBoulders/AudibleBoulders
# Refactor usersDashboards request handler

#### What's this PR do?
1) Refactored userDashboards request handler. Queries are no longer in the request handler. Queries are separated in the queries folder under src/server/queries. Queries are promisified. I utilize this in my request handler to implement the handleDelete function in userDashboards request handler.

#### What are the important parts of the code?
1) src/server/new-request-handlers/usersDashboards.js
2) src/server/queries/diffs.js (line 57, missing * in select statement)

#### How should this be tested by the reviewer?
1) Pull down
2) Start node server
3) Make sure you have data in all tables, including diffs.
4) Use postman to make a DELETE request to the following API endpoint: http://localhost:8080/api/users_dashboards/:githubId/:dashboardId
5) Verify that the diffs are removed for the specified user.
6) Verify that the user is removed from the users_dashboards table.

#### Is any other information necessary to understand this?
N/A

#### What are the relevant tickets? (Please add `closes`, `refs`, etc)
N/A

#### Screenshots (if appropriate)
N/A
